client/my-sites/email/email-providers-comparison/in-depth/data.tsx

```diff
@@ -45,10 +54,22 @@ export const googleWorkspaceFeatures: EmailProviderFeatures = {
 		/>
 	),
 	list: {
+		collaboration: translate( 'Real-time collaboration for Docs, Sheets, and Slides' ),
 		importing: translate( 'Easy to import your existing emails and contacts' ),
-		storage: translate( '30GB storage' ),
+		storage: translate( '30GB storage for emails and cloud storage' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( '30GB or unlimited cloud storage' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/copy-in-email-comparison-pages)